lib/addons/index.ts

```diff
@@ -5,6 +5,7 @@ export { CalicoAddOn } from './calico'
 export { ContainerInsightsAddOn } from './container-insights'
 export { ClusterAutoScalerAddOn } from './cluster-autoscaler'
 export { ExternalDnsAddon } from './external-dns'
+export { OpaAddOn } from './OPA'
```

Comment:
OPA directory should be rename to opa lower-case (consistent naming convention). E.g. dns, nginx, etc. all lower case.

Reply:
addressed